src/DebugApi.ts

```diff
@@ -277,8 +277,8 @@ export class DebugApi {
         // wait 3s to pool responses
         return Bluebird.delay(3000).then(function() {
             // unhook listener to avoid leaking
-            if (client.unsafeClient) {
-                client.unsafeClient.removeListener("raw", listener);
+            if (client.state.status === BridgedClientStatus.CONNECTED) {
```

Comment:
`client.state.status` feels verbose. We should probably at least expose `get status()` on the client.

Reply:
i am not sure that typescript will allow access to `client.state.client` then

Reply:
To be honest I've never been happy with bits of the bridge accessing unsafeClient to begin with. I've switched it around so that BridgedClient exposes methods for each of these calls.